### PR TITLE
Utility Pole Support

### PR DESCRIPTION
**Pull request by srothst1**
This PR adds support for utility poles.

cc: https://github.com/3DStreet/3dstreet-assets/issues/12

### CODE REVIEW

File: src/aframe-streetmix-parsers.js [hunk 1]
```diff
@@ -149,9 +150,9 @@ function createDividerVariant (variantName, positionX, clonedObjectRadius, step
   return dividerParentEl;
 }
 
-function createClonedVariants (variantName, positionX, clonedObjectRadius, step = 2.25) {
+function createClonedVariants (variantName, positionX, clonedObjectRadius, step = 2.25, rotation = '0, 0, 0') {
```
Comment (kfarr) [inline src/aframe-streetmix-parsers.js:153]: pretty sure this should be `rotation = '0 0 0'`

**Comment by kfarr** [comment]
also the pole should probably be twice as high.

Compare:
image

Vs:
image

**Review by kfarr — COMMENTED** [review]
- would be good to get @gnansai update to the model itself, probably 2x (see PR comments)
- also need to run `npm run dist` to build the dist js file and commit that

File: src/aframe-streetmix-parsers.js (hunk 1 above)
Comment (srothst1) [reply src/aframe-streetmix-parsers.js]: Nice catch :+1: I just added a commit that addresses this.

**Comment by srothst1** [comment]
@kfarr I agree 100% with your feedback. The scale of the utility pole is not quite right.

**Comment by kfarr** [comment]
> @kfarr I agree 100% with your feedback. The scale of the utility pole is not quite right.

Here's an example at 2x scale, what do you think?
image

**Review by kfarr — CHANGES_REQUESTED** [review]

**Comment by gnansai** [comment]
https://github.com/gnansai/3dstreet-assets/tree/main/sets/street-props/gltf-exports
I scaled up the utility-pole 2 times

File: src/assets.js [hunk 2]
```diff
@@ -20,6 +20,7 @@
           <a-mixin id="outdoor_dining" gltf-part-plus="src: #streetProps; part: outdoor_dining"></a-mixin>
           <a-mixin id="bench_orientation_center" gltf-part-plus="src: #streetProps; part: bench_orientation_center"></a-mixin>
           <a-mixin id="parklet" gltf-part-plus="src: #streetProps; part: parklet"></a-mixin>
+          <a-mixin id="utility_pole" gltf-part-plus="src: #streetProps; part: utility_pole"></a-mixin>
```
[the remark this answers is not in the document]
Comment (kfarr) [reply src/assets.js:23]: Nevermind, don't do this and instead just used gnanasai's updated version that he just added and I merged to https://github.com/3DStreet/3dstreet-assets/pull/17/commits/0a6d78b224302a6d6984138c945e478238232d99

[the remark this answers is not in the document]
Comment (srothst1) [reply src/assets.js:23]: Sounds good

**Comment by srothst1** [comment]
@kfarr @gnansai we should be good to go :+1:

**Review by kfarr — APPROVED** [review]

**Pull request merged**